voicevox_engine/engine_manifest/EngineManifest.py

```diff
@@ -55,4 +55,5 @@ class EngineManifest(BaseModel):
     terms_of_service: str = Field(title="エンジンの利用規約")
     update_infos: List[UpdateInfo] = Field(title="エンジンのアップデート情報")
     dependency_licenses: List[LicenseInfo] = Field(title="依存関係のライセンス情報")
+    supported_vvlib_manifest_version: str = Field(title="エンジンが対応するvvlibのバージョン")
```

Comment:
manifestじゃないvvlibのバージョンは現れないので、文字数削減のために消しちゃっても良いかも。
```suggestion
    supported_vvlib_version: str = Field(title="エンジンが対応するvvlibのバージョン")
```

Reply:
音声ライブラリ自体にもバージョンを設定できるようにしている(例えば、音声ライブラリのモデルファイルや、話者関連情報を更新したい場合)ので、そのようにしていいか若干微妙な気がしています(もちろん、エンジンで確認するのは`manifest_version`だけなのですが、一応混同しないように`manifest_version`にしています)
https://github.com/y-chan/voicevox_engine/blob/039a2b00c026f564377b543acd1db9aa56423bce/voicevox_engine/model.py#L299

Reply:
あーなるほです！！　manifest_versionのが良いなと思いました！